tests/test_utils_url.py

```diff
@@ -232,13 +225,11 @@ def do_expected(self):
     t_method.__name__ = 'test_uri_%03d' % k
     setattr (GuessSchemeTest, t_method.__name__, t_method)
 
-# TODO: the following tests do not pass with current implementation
-for k, args in enumerate([
+for j, args in enumerate([
             (r'C:\absolute\path\to\a\file.html', 'file://',
              'Windows filepath are not supported for scrapy shell'),
-        ], start=1):
-    t_method = create_skipped_scheme_t(args)
-    t_method.__name__ = 'test_uri_skipped_%03d' % k
+        ], start=k+1):
+    t_method.__name__ = 'test_uri_%03d' % j
```

Comment:
You are not calling `t_method = create_guess_scheme_t(args)` here, so `t_method` must come from the previous loop. Does it work it you replace `create_skipped_scheme_t` by `create_guess_scheme_t` instead of simply removing that statement?

Reply:
Actually it is failing that way. So it was actually re-running the last test and then showing 'PASSED' . Btw are there any plans on making this work?

Reply:
[As far as I can tell](https://github.com/scrapy/scrapy/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+windows+scheme) there is not open bug report about it. Feel free to create one, we should definitely not forget about this, and it’s probably not that hard to fix.